### PR TITLE
feat: Add Sponsor Logos

### PR DESCRIPTION
**Pull request by padmajabhol**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by riteshsp2000** [comment]
@ORKO06 @padmajabhol Did you not check the logos. How will a black colored text logo be visible on black background? Just directly updated with the sponsor images without any checks?

**Comment by riteshsp2000** [comment]
Few changes which are necessary. 
1. All logos must be of similar sizes. Even though we have same attributes (css styles) for all the logos, due to the various aspect ratios of the logos they appear to be small or large. 
- To maintain a consistent size, in the sponsor object (inside the array) add one more property called width. 
- Use this property in the jsx to set the width dynamically. For all the logos that appear huge, set width of a lesser value and for others a higher value. 
- NOTE: set width in terms of percentages and not pixels. Though keep a minimum width for each logo for smaller width screens
- Following are the logos whose size needs to be reduced.
a. Diamond Jubilee
b. Leading Learners

2. From the design team get the following black logos converted to white.
- Polygon
- Linode

3. For all the logos that have a box like structure in the background, remove the background. Get it done either from Design team or you can use this tool [Remove.bg](https://remove.bg). Following is the logo whose background needs to be removed.
- Celo

4. The margin between the different sponsor categories is not much. It feels like all the sponsors are crammed into one. Increase the padding. Talk to the Design team and see if the category names could be highlighted in some or the other way.

5. In the footer, I have updated the logos of all the clubs with higher resolution images. Since lower resolution images were used, no hard coded width was set which is not at all ideal. Set a width for all the logos.

File: config/content/Hero.js [hunk 1]
```diff
@@ -14,13 +14,41 @@ export default {
...
-    { id: 8, src: '/hero-icons/tesla.svg', alt: 'tesla logo' },
+    {
+      id: 1,
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/alpha/vlnvmirmr4r7tolmdrxt.png',
+      alt: 'Devfolio',
+      size: '150px',
+    },
+    {
+      id: 2,
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/alpha/oigy7pgs2haeoh8dtqon.png',
```
Comment (DesignrKnight) [inline config/content/Hero.js:25]: GitHub Logo is blurred at edges

File: config/content/Hero.js [hunk 2]
```diff
@@ -14,13 +14,41 @@ export default {
...
+      alt: 'Github',
+      size: '120px',
+    },
+    {
+      id: 3,
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/alpha/ogaeyl6batos4n5cumtz.png',
```
Comment (DesignrKnight) [inline config/content/Hero.js:31]: Very large size image. Use squoosh.app

File: config/content/Sponsors.js [hunk 3]
```diff
@@ -5,162 +5,85 @@ export default {
...
-      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/kmuaxvxjtlgl7a7w5jnw.png',
-      name: 'sponsor-1',
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/gamma/vgblcn40kjajxwxmph68.png',
+      name: 'Taskade',
+      size: '95%',
     },
     {
-      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/kmuaxvxjtlgl7a7w5jnw.png',
-      name: 'sponsor-1',
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/gamma/hendtn2wfa16wyuxk7so.png',
```
Comment (DesignrKnight) [inline config/content/Sponsors.js:52]: Even this is a huge image

File: config/content/Sponsors.js [hunk 4]
```diff
@@ -5,162 +5,85 @@ export default {
...
-      name: 'sponsor-1',
-    },
-    {
-      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/kmuaxvxjtlgl7a7w5jnw.png',
-      name: 'sponsor-1',
-    },
-    {
-      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/kmuaxvxjtlgl7a7w5jnw.png',
-      name: 'sponsor-1',
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/delta/aksbtyjd4h8ekwtlwt2u.png',
```
Comment (DesignrKnight) [inline config/content/Sponsors.js:64]: Again large image

File: config/content/Sponsors.js [hunk 5]
```diff
@@ -5,162 +5,85 @@ export default {
...
-      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/kmuaxvxjtlgl7a7w5jnw.png',
-      name: 'sponsor-1',
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/delta/jcr1qdx3oivszgrpxqml.png',
+      name: 'Linode',
+      size: '95%',
     },
     {
-      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/kmuaxvxjtlgl7a7w5jnw.png',
-      name: 'sponsor-1',
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/delta/vs6j6vhnblodwddpnxeu.png',
```
Comment (DesignrKnight) [inline config/content/Sponsors.js:74]: Image large

**Review by DesignrKnight — COMMENTED** [review]
@CIPHERTron 
Refactor code to import sponsors in a common file and refer that in ticker and sponsor section instead of manually adding same thing at two places

**Comment by DesignrKnight** [comment]
Ticker Issues
1. Logos are not centred
2. Height is computed based on child elements, it should be fixed and not dependent on children. You can see the height initially is small and then increases as logos load

**Comment by CIPHERTron** [comment]
> Ticker Issues
> 
> 1. Logos are not centred
> 2. Height is computed based on child elements, it should be fixed and not dependent on children. You can see the height initially is small and then increases as logos load

Fixed!!

**Comment by CIPHERTron** [comment]
> @CIPHERTron
> Refactor code to import sponsors in a common file and refer that in ticker and sponsor section instead of manually adding same thing at two places

Refactor done!!

File: config/content/Hero.js (hunk 2 above)
Comment (CIPHERTron) [reply config/content/Hero.js]: > Very large size image. Use squoosh.app

Yeah, I'll squoosh the large sized logos and update it in cloudinary🚀
Thanks for reviewing it @DesignrKnight <3

**Review by CIPHERTron — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]

**Pull request merged**